bootstrap.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+TLA=${LIGHTUP_TLA}
+TOK=${LIGHTUP_TOKEN}
```

Comment:
Why not just use LIGHTUP_TLA and LIGHTUP_TOKEN?

Reply:
:)

Reply:
I'll modify it.

Reply:
actually, i left it as is. it forces the script to start out with picking up those values. if you end up invoking it without those values in the environment, it immediately complains that variables are unbound and then exits.